### ISSUE TITLE
Addon stopped working without vite

### ISSUE THREAD

**Issue by SenseiMarv** [UX-and-I/storybook-design-token#128]
# Current behaviour
Currently, upgrading to version `2.7.0` of the addon breaks its functionality when not adding additional dependencies. The console will show `Error: Cannot find module 'vite'` when starting Storybook:

```bash
❯ y storybook
yarn run v1.22.19
$ start-storybook -p 8010
info @storybook/react v6.5.12
info
info => Loading presets
WARN   Failed to load preset: {"type":"presets","name":"/Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/storybook-design-token/dist/preset.js"} on level 1
ERR! Error: Cannot find module 'vite'
ERR! Require stack:
ERR! - /Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/storybook-design-token/dist/preset.js
ERR! - /Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/core-common/dist/cjs/presets.js
ERR! - /Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/core-common/dist/cjs/index.js
ERR! - /Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/core-server/dist/cjs/index.js
ERR! - /Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/react/node_modules/@storybook/core/dist/cjs/server.js
ERR! - /Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/react/node_modules/@storybook/core/server.js
ERR! - /Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/react/dist/cjs/server/index.js
ERR! - /Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/react/bin/index.js
ERR!     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
ERR!     at Function.Module._load (node:internal/modules/cjs/loader:778:27)
ERR!     at Module.require (node:internal/modules/cjs/loader:1005:19)
ERR!     at require (node:internal/modules/cjs/helpers:102:18)
ERR!     at Object.<anonymous> (/Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/storybook-design-token/dist/preset.js:50:19)
ERR!     at Module._compile (node:internal/modules/cjs/loader:1105:14)
ERR!     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
ERR!     at Module.load (node:internal/modules/cjs/loader:981:32)
ERR!     at Function.Module._load (node:internal/modules/cjs/loader:822:12)
ERR!     at Module.require (node:internal/modules/cjs/loader:1005:19)
ERR!  Error: Cannot find module 'vite'
ERR! Require stack:
ERR! - /Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/storybook-design-token/dist/preset.js
ERR! - /Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/core-common/dist/cjs/presets.js
ERR! - /Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/core-common/dist/cjs/index.js
ERR! - /Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/core-server/dist/cjs/index.js
ERR! - /Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/react/node_modules/@storybook/core/dist/cjs/server.js
ERR! - /Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/react/node_modules/@storybook/core/server.js
ERR! - /Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/react/dist/cjs/server/index.js
ERR! - /Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/react/bin/index.js
ERR!     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
ERR!     at Function.Module._load (node:internal/modules/cjs/loader:778:27)
ERR!     at Module.require (node:internal/modules/cjs/loader:1005:19)
ERR!     at require (node:internal/modules/cjs/helpers:102:18)
ERR!     at Object.<anonymous> (/Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/storybook-design-token/dist/preset.js:50:19)
ERR!     at Module._compile (node:internal/modules/cjs/loader:1105:14)
ERR!     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
ERR!     at Module.load (node:internal/modules/cjs/loader:981:32)
ERR!     at Function.Module._load (node:internal/modules/cjs/loader:822:12)
ERR!     at Module.require (node:internal/modules/cjs/loader:1005:19) {
ERR!   code: 'MODULE_NOT_FOUND',
ERR!   requireStack: [
ERR!     '/Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/storybook-design-token/dist/preset.js',
ERR!     '/Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/core-common/dist/cjs/presets.js',
ERR!     '/Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/core-common/dist/cjs/index.js',
ERR!     '/Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/core-server/dist/cjs/index.js',
ERR!     '/Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/react/node_modules/@storybook/core/dist/cjs/server.js',
ERR!     '/Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/react/node_modules/@storybook/core/server.js',
ERR!     '/Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/react/dist/cjs/server/index.js',
ERR!     '/Users/mstickel/projects/beyond/ng-merchant-ui/node_modules/@storybook/react/bin/index.js'
ERR!   ]
ERR! }
info Addon-docs: using MDX1
info => Using implicit CSS loaders
info => Using default Webpack5 setup
```

Storybook is still able to start up, but the design tokens will not work and the browser shows errors while trying to load `design-tokens.source.json`:

![image](https://user-images.githubusercontent.com/41735601/192996702-92bcbf5f-36bb-4db6-96f1-d5dedf843812.png)

Manually adding `vite` as dependency (`yarn add -D vite`) fixes the issue.

# Expected behaviour
It should either be communicated in the documentation that `vite` is a required dependency for this addon to work, or it should become optional to have `vite` installed. Since Storybook will work with both Webpack or Vite, I would argue that it should be optional.

# Steps to reproduce
1. Have Storybook installed and set up with Webpack
2. Add this addon in version `2.7.0`
3. Start Storybook (`npm run storybook`/ `yarn storybook`)
4. Observe the error on the console
5. Observe the addon not working and the reported error in the browser

# Hints
When looking into the lock file, I can see that `vite` is not listed as peer dependency, which it should be if this addon relies on it to work:
```
storybook-design-token@2.7.0:
  version "2.7.0"
  resolved "https://registry.yarnpkg.com/storybook-design-token/-/storybook-design-token-2.7.0.tgz#5770a70485504a051bb4186eed6f1def41ef0ad4"
  integrity sha512-E24ZDOBz+Y60aZqJPLp0b2+dDWPbjUHk3h+lqLiWNg6ehnEba0d0RkOFz8uxiF1uQShg8NpMlHjIcgYlwTK3ug==
  dependencies:
    "@storybook/addons" "^6.3.12"
    "@storybook/components" "^6.3.12"
    "@storybook/theming" "^6.3.12"
    jsdom "^19.0.0"
    polished "^4.1.3"
    postcss "^8.3.11"
    postcss-scss "^4.0.2"
    raw-loader "^4.0.2"
    react-storage-hooks "^4.0.1"
    react-use-clipboard "^1.0.7"
    react-virtual "^2.10.4"
```

**Comment by Sqrrl** [comment]
Sorry for that. Should be fixed with v2.7.1

**Comment by SenseiMarv** [comment]
Sorry for taking a bit longer to reply. I tried out whether the newest version runs without Vite and can confirm that it now starts up again 👍
